pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -767,8 +767,12 @@ stdenvNoCC.mkDerivation {
       export hardening_unsupported_flags="${concatStringsSep " " ccHardeningUnsupportedFlags}"
     ''
 
+    + optionalString (!targetPlatform.isx86_32) ''
```

Comment:
Could the reason for this check please be explained in a comment?

Reply:
Maybe not just a comment. Maybe "should enable frame pointers by default" should go in `lib.platforms`?

Reply:
Just checking because I don't know the architecture specifics: this reasoning applies only to x86, and not to other 32-bit architectures?